FiveCalls/FiveCalls/MyImpactViewController.swift

```diff
@@ -61,31 +62,22 @@ class MyImpactViewController : UITableViewController {
             NSAttributedStringKey.foregroundColor: UIColor.white
         ]
         
-        viewModel = ImpactViewModel(logs: ContactLogs.load().all)
-        let weeklyStreakCount = viewModel.weeklyStreakCount
-        streakLabel.text = weeklyStreakMessage(for: weeklyStreakCount)
-        let numberOfCalls = viewModel.numberOfCalls
-        impactLabel.text = impactMessage(for: numberOfCalls)
-        if numberOfCalls == 0 {
-            subheadLabel.isHidden = true
-            subheadLabel.addConstraint(NSLayoutConstraint(item: subheadLabel, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 0))
-        }
+        updateStats()
         
         profilePic.layer.borderColor = UIColor.white.cgColor
         profilePic.layer.borderWidth = 2.0
 
         sizeHeaderToFit()
         
-        let op = FetchStatsOperation()
-        op.completionBlock = {
-            self.totalCalls = op.numberOfCalls
+        let totalCallsOp = FetchStatsOperation()
+        totalCallsOp.completionBlock = {
+            self.totalCalls = totalCallsOp.numberOfCalls
             DispatchQueue.main.async {
                 self.tableView.reloadData()    
             }
-            
         }
-        OperationQueue.main.addOperation(op)
-        
+        OperationQueue.main.addOperation(totalCallsOp)
```

Comment:
Why main queue?

Reply:
This wasn't a conscious choice on my part, I was just following the established logic used by the existing Operations.  It looks like we do mark our Operations as asynchronous, and their logic operates via callbacks, so I'm not sure it matters very much, but we could certainly create another queue for this if you want.